Server/ReasnAPI/ReasnAPI/Services/ImageService.cs

```diff
@@ -97,15 +97,15 @@ public void UpdateImagesForEvent(int objectId, List<ImageDto> imageDtos)
         context.SaveChanges();
     }
 
-    public void UpdateImage(int objectId, ImageDto imageDto)
+    public void UpdateImageForUser(int userId, ImageDto imageDto)
     {
         if (imageDto is null)
         {
             throw new ArgumentException("No image provided");
         }
 
-        var image = context.Images.Where(i => i.ObjectType == imageDto.ObjectType).FirstOrDefault(r => r.ObjectId == objectId);
-        if (image is null)
+        var image = context.Images.FirstOrDefault(i => i.ObjectType == ObjectType.User && i.ObjectId == userId);
+        if (image == null)

```

Comment:
Why changed from `is` to `==`?

Reply:
i removed both lines and writed again didnt notice that, changed